libraries/apollo-tooling/src/test/kotlin/com/apollographql/apollo3/tooling/IntrospectionQueryTest.kt

```diff
@@ -99,115 +99,16 @@ class IntrospectionQueryTest {
       }
     }""".trimIndent()
 
-  private val normalizedV3IntrospectionQuery = """
-    query IntrospectionQuery {
-      __schema {
-        queryType {
-          name
-        }
-        mutationType {
-          name
-        }
-        subscriptionType {
-          name
-        }
-        types {
-          ...FullType
-        }
-        directives {
-          name
-          description
-          locations
-          args {
-            ...InputValue
-          }
-        }
-      }
-    }
-
-    fragment FullType on __Type {
-      kind
-      name
-      description
-      fields(includeDeprecated: true) {
-        name
-        description
-        args {
-          ...InputValue
-        }
-        type {
-          ...TypeRef
-        }
-        isDeprecated
-        deprecationReason
-      }
-      inputFields {
-        ...InputValue
-      }
-      interfaces {
-        ...TypeRef
-      }
-      enumValues(includeDeprecated: true) {
-        name
-        description
-        isDeprecated
-        deprecationReason
-      }
-      possibleTypes {
-        ...TypeRef
-      }
-    }
-
-    fragment InputValue on __InputValue {
-      name
-      description
-      type {
-        ...TypeRef
-      }
-      defaultValue
-    }
-
-    fragment TypeRef on __Type {
-      kind
-      name
-      ofType {
-        kind
-        name
-        ofType {
-          kind
-          name
-          ofType {
-            kind
-            name
-            ofType {
-              kind
-              name
-              ofType {
-                kind
-                name
-                ofType {
-                  kind
-                  name
-                  ofType {
-                    kind
-                    name
-                  }
-                }
-              }
-            }
-          }
-        }
-      }
-    }""".trimIndent()
-
-  private fun String.normalized(): String {
+  private fun String.asParsedGQLDocument(): String {
     return Buffer().writeUtf8(this).use {
       it.parseAsGQLDocument(null).getOrThrow().toUtf8().trimIndent()
     }
   }
 
+  private fun String.normalized() = this.trimIndent().replace(Regex("\\s+"), " ")
+
   @Test
   fun canUseTheSameIntrospectionQueryAsVersion3() {
-    assertEquals(normalizedV3IntrospectionQuery, v3IntrospectionQuery.normalized())
+    assertEquals(v3IntrospectionQuery.normalized(), v3IntrospectionQuery.asParsedGQLDocument().normalized())
```

Comment:
I'm confused. this compares v3 to v3. Shouldn't it include `IntrospectionQuery.QUERY_DOCUMENT` somewhere?

Reply:
Yeah this tests that the v3 query can still be parsed but that's not very useful. If we compare the v3 introspection query to IntrospectionQuery.OPERATION_DOCUMENT, they're a bit different though (order of the fragments + some added `__typename`).

Reply:
The test should make more sense now 😅. The v3 query is the same as the June2018 after normalization.